### PR TITLE
Add MSI installer

### LINKED ISSUE #14: Provide MSI install package
The wrapper is incredibly helpful, especially for system administrators. Therefore it would be great to be able to roll it out via GPO software deployment, but for that a MSI package would be needed.

### PR DESCRIPTION
**Pull request by galeksandrp**
fixes #14

For compilation install [WiX Toolset](http://wixtoolset.org/).
```batch
"%ProgramFiles(x86)%\WiX Toolset v3.10\bin\candle" RDPWInst.wxs
"%ProgramFiles(x86)%\WiX Toolset v3.10\bin\light" RDPWInst.wixobj
```

### CODE REVIEW

File: RDPWInst.wxs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          <Directory Id="INSTALLLOCATION" Name="{$(var.ProductCode)}">
+            <Component Id="ProductComponent" Guid="affd77d1-b35c-46f3-a97f-1686dc57b8b8">
+              <File Id='RDPWInst' DiskId='1' Source='RDPWInst.exe'/>
+            </Component>
+          </Directory>
+        </Directory>
+      </Directory>
+    </Directory>
+
+    <Feature Id="ProductFeature" Title="SetupProject1" Level="1">
```
Comment (binarymaster) [inline RDPWInst.wxs]: These lines looks like a template, it would be better to give them proper names

File: RDPWInst.wxs [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+  <?define ProductName="RDP Wrapper Library" ?>
+  <?define ProductVersion="1.6.1" ?>
+  <?define ProductCode="37ea5771-3352-4a52-9fac-9297331daebd"?>
```
Comment (binarymaster) [inline RDPWInst.wxs]: Just wondering where these GUIDs came from?

File: RDPWInst.wxs [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    <Property Id="DISABLEROLLBACK" Value="1" />
+    <MajorUpgrade AllowDowngrades="yes"/>
+
+    <Media Id="1" Cabinet="media1.cab" EmbedCab="yes"/>
+
+    <Directory Id="TARGETDIR" Name="SourceDir">
+      <Directory Id="CommonAppDataFolder">
+        <Directory Id="PACKAGECACHE" Name="Package Cache">
+          <Directory Id="INSTALLLOCATION" Name="{$(var.ProductCode)}">
+            <Component Id="ProductComponent" Guid="affd77d1-b35c-46f3-a97f-1686dc57b8b8">
```
Comment (binarymaster) [inline RDPWInst.wxs]: And this

File: RDPWInst.wxs [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+    <Feature Id="ProductFeature" Title="SetupProject1" Level="1">
+      <ComponentRef Id="ProductComponent" />
+    </Feature>
+
+    <CustomAction Id='InstallAction' FileKey='RDPWInst' ExeCommand='-i -o' Execute='immediate' Return='check'/>
+    <CustomAction Id='UninstallAction' FileKey='RDPWInst' ExeCommand='-u' Execute='immediate' Return='check'/>
+    <CustomAction Id='UpdateAction' FileKey='RDPWInst' ExeCommand='-w' Execute='immediate' Return='check'/>
+<!--     <CustomAction Id='ChangeAction' Directory='ProgramFilesFolder' ExeCommand='RDP Wrapper\RDPConf' Execute='immediate' Return='check'/>
+    <CustomAction Id='RepairAction' Directory='ProgramFilesFolder' ExeCommand='RDP Wrapper\RDPCheck' Execute='immediate' Return='check'/> -->
```
Comment (binarymaster) [inline RDPWInst.wxs]: For repair action, does it possible to execute `RDPWInst` twice to run uninstall and then install it again?

File: RDPWInst.wxs (hunk 2 above)
Comment (galeksandrp) [inline RDPWInst.wxs]: Autogenerated from https://www.guidgen.com/ as recommended by [WiX Documentation](http://wixtoolset.org/documentation/manual/v3/howtos/general/generate_guids.html).

File: RDPWInst.wxs (hunk 3 above)
Comment (galeksandrp) [inline RDPWInst.wxs]: Autogenerated from https://www.guidgen.com/ as recommended by [WiX Documentation](http://wixtoolset.org/documentation/manual/v3/howtos/general/generate_guids.html).

**Comment by galeksandrp** [comment]
Install [WiX Toolset](http://wixtoolset.org/).
```batch
"%ProgramFiles(x86)%\WiX Toolset v3.10\bin\candle" RDPWInst.wxs
"%ProgramFiles(x86)%\WiX Toolset v3.10\bin\light" RDPWInst.wxs
```

File: RDPWInst.wxs (hunk 4 above)
Comment (binarymaster) [inline RDPWInst.wxs]: Any news?

**Pull request merged**